packages/vue/README.md

```diff
@@ -10,7 +10,7 @@ Assuming you have a Vue 3 project, just run:
 npm install --save @starport/vue
 ```
 
-The components also require `@starport/vuex` to be installed which is set as a peer dependency. More information [here](https://github.com/tendermint/vue/tree/develop/packages/vuex)
+The components also require `@starport/vuex` to be installed that is set as a peer dependency. For more information, see [packages/vuex](https://github.com/ignite-hq/web/tree/develop/packages/vuex).
```

Comment:
I preserved all `@starport/vuex` mentions

Reply:
Thank you! Soon enough we'll not have them anymore.